scRNA-seq-advanced/setup/SCPCP000005-samples/config.yaml

```diff
@@ -0,0 +1,8 @@
+# The main directory where processed files will be stored
+base_dir: /shared/data/training-modules/scRNA-seq-advanced/data/SCPCP000005-samples
+
+# Libraries to process
+libraries: ["SCPCL000478", "SCPCL000479", "SCPCL000480", "SCPCL000481"]
+
+# Celltypes TSV file
+celltypes_file: annotations/celltypes.tsv
```

Comment:
This should probably be stored in the data directory rather than in this repo. I suggested a location: 
```suggestion
celltypes_file: /shared/data/training-modules/scRNA-seq-advanced/data/SCPCP000005-samples/annotations/celltypes.tsv
```

Note that this location is already in the linked files.